examples/mosviz-example.ipynb

```diff
@@ -2,12 +2,25 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```

Comment:
Make sure you clear the output from the cells before committing the notebook (maybe just add a new commit with the output clearerd)